### PR TITLE
Bump up dependencies

### PR DESCRIPTION
**Pull request by ochikov**
Signed-off-by: ochikov <ognyan@limechain.tech>

**Description**:
Bump up dependencies

**Related issue(s)**:

Fixes #

**Notes for reviewer**:
In order to pass the Github actions, we need to first do the following steps:
- **prepare and release a beta release for the proto package**
- **prepare and release a beta release for the cryptography package**
- **use the beta versions in the SDK**
- **observe for any build/test errors**

Tested **locally** with linked packages and the SDK is built successfully. Unit & Integration tests run successfully.
We can do the above actions and prepare new beta SDK release.
Please Note: Some dependencies cannot be updated as the `long` package for example. If we update it, we need to fix protobufs types as well.

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by ochikov** [comment]
@steven-sheehy I will be happy for your thoughts on the comments above.

File: src/account/AccountUpdateTransaction.js [hunk 1]
```diff
@@ -265,9 +265,7 @@ export default class AccountUpdateTransaction extends Transaction {
                     update.stakedNodeId != null
                         ? update.stakedNodeId
                         : undefined,
-                declineStakingReward:
-                    update.declineReward != null &&
-                    update.declineReward == true,
+                declineStakingReward: update.declineReward != null,
```
Comment (petreze) [inline src/account/AccountUpdateTransaction.js:268]: Why we are removing the following check `update.declineReward == true`?

File: src/contract/ContractUpdateTransaction.js [hunk 2]
```diff
@@ -279,9 +279,7 @@ export default class ContractUpdateTransaction extends Transaction {
                     update.stakedNodeId != null
                         ? update.stakedNodeId
                         : undefined,
-                declineStakingReward:
-                    update.declineReward != null &&
-                    update.declineReward == true,
+                declineStakingReward: update.declineReward != null,
```
Comment (petreze) [inline src/contract/ContractUpdateTransaction.js:282]: Same here. Is it unnecessary to check if `update.declineReward == true`?

File: src/account/AccountUpdateTransaction.js (hunk 1 above)
Comment (ochikov) [reply src/account/AccountUpdateTransaction.js:268]: Due to this error:
Screenshot 2023-01-20 at 18 44 51

I am still figuring it out.

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by petreze — APPROVED** [review]

**Pull request merged**